Boss/Mod/InitialRebalancer.cpp

```diff
@@ -182,11 +182,6 @@ class InitialRebalancer::Impl::Run::Impl
 
 					auto cs = p["channels"];
```

Comment:
if a user run `deprecated-api = false`, this object is null

from cln docs of listpeers: `channels (array of objects, optional) deprecated in v23.02, removed after v24.02`